packages/terra-show-hide/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Changed
+  * Updated WDIO Screenshots due to added focus.
```

Comment:
I feel like this item isn't relevant to consumers and can be removed

Reply:
Good catch, removed in [this commit](https://github.com/cerner/terra-core/pull/3830/commits/9e8aff2a397aa02be6fda129d3f049612e78d1e0)